src/renderer/pages/Staking/Overview/Overview.tsx

```diff
@@ -169,6 +180,22 @@ export const Overview = () => {
     return acc;
   }, []);
 
+  const [selectedValidators, notSelectedValidators] = nominators.reduce<[Validator[], Validator[]]>(
+    (acc, nominator) => {
+      if (validators[nominator.address]) {
+        acc[0].push({
```

Comment:
what do you think to replace acc indexes with constants or use object instead of array?

Reply:
Object is more convinient than array but I don't see the need to replace it here, easier to work with arrays as input for ValidatorsModal component